ChangeLog.md

```diff
@@ -4,6 +4,11 @@ The format is based on
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v3.0.1 -- YYYY-MM-DD
+
+### Removed
+* `freud.order.Translational`, should have been removed with v3.0.0.
```

Comment:
```suggestion
* `freud.order.Translational`
```
Not sure if this is necessary.

Reply:
It's fine if its not in the changelog, but I'd like to make a note that it should have been removed with v3 somewhere.